src/Compilers/VisualBasic/Test/Syntax/Parser/ParseErrorTests.vb

```diff
@@ -1739,6 +1739,18 @@ BC30306: Array subscript expression missing.
                              </errors>)
     End Sub
 
+    <Fact()>
+    Public Sub BC30241ERR_ExpectedNamedArgument_VB15_6()
+        Dim code = <![CDATA[
+<Attr1(1, b:=2, 3, e:="Scen1")>
+Class Class1
+
+End Class
+            ]]>.Value
+
+        ParseAndVerify(code, LanguageVersion.VisualBasic15_6, False, Nothing) ' No parsing errors
```

Comment:
>' No parsing errors [](start = 78, length = 19)

This doesn't feel right, it looks like names in attributes refer to members rather than to parameters.